src/idd/idd.utils.js

```diff
@@ -235,6 +235,43 @@ InteractiveDataDisplay.Utils =
             return { minx: minx, maxx: maxx, miny: miny, maxy: maxy };
         },
 
+        enumPlots: function (masterPlot) {
```

Comment:
@ElenaPochernina There're no recursive call of enum for children of children:
enumPlots: function(p) {
                  var masterPlot = p.master;
                   var plotsArray = [];
 +            var enumRec = function (p, plotsArray) {
 +                plotsArray.push(p);
                    if (masterPlot.children)
 +                   masterPlot.children.forEach(function (child) {
 +                      enumRec(child, plotsArray);
 +                   });
 +            };
 +            enumRec(masterPlot);
 +            plotsArray.sort(function (a, b) { return b.order - a.order; });
 +            return plotsArray;